### PR TITLE
Improve a few strings in the new terraforming missions

### PR DESCRIPTION
**Pull request by Pointedstick**
The new terraforming missions are awesome! While playing through them I noticed a few instances awkwardly-worded text. This PR attempts to improve them.

### CODE REVIEW

File: data/transport missions.txt [hunk 1]
```diff
@@ -2280,7 +2280,7 @@ mission "Terraforming 9"
 	
 	on complete
 		dialog 
-			`The journey to Tundra was spent by your passengers discussing the possible methods to terraform the planet. The Republic representative was none too pleased when Nolan, perhaps jokingly, suggested that we crash an asteroid into the planet.` 
+			`The journey to Tundra was spent by your passengers discussing the possible methods of terraforming the planet. The Republic representative was none too pleased when Nolan, perhaps jokingly, suggested crashing an asteroid into it.`
```
Comment (Bladewood) [inline data/transport missions.txt]: I agree with the changes to the crashing part of this line, although I don't think the terraforming part was necessary. The only thing that annoys me there is "the" before "possible methods".

File: data/transport missions.txt [hunk 2]
```diff
@@ -2299,7 +2299,7 @@ mission "Terraforming 10"
...
-			`	"Tundra has very low volcanic activity, but seismic mapping has lead us to this area." Amy circles the red spot with her finger. "Magma from the planet's mantle is building up under this area of the crust, which lucky for us is considerably thin. Still, it could be another ten thousand years before the magma builds up enough pressure under the surface to cause a supereruption, melting the snow around it and releasing greenhouse gases into the atmosphere that over time will raise the global temperatures."`
+			`	"Tundra has very low volcanic activity, but seismic mapping has lead us to this area." Amy circles the red spot with her finger. "Magma from the planet's mantle is building up under this area of the crust, which lucky for us is unusually thin. Still, it could be another ten thousand years before the magma builds up enough pressure under the surface to cause a supereruption, melting the snow around it and releasing greenhouse gases into the atmosphere that over time will raise the global temperatures."`
```
Comment (Bladewood) [inline data/transport missions.txt]: I don't think this change is necessary.

Comment (Zireael07) [reply data/transport missions.txt]: As an English major, I agree with @Pointedstick, "considerably thin" is weird. "Very thin" or "extremely thin" are definitely better.

**Comment by Amazinite** [comment]
The changes look good to me. Glad you enjoyed the missions. :)

**Pull request merged**